channels/microsoft365/Microsoft365errors.rst

```diff
@@ -0,0 +1,151 @@
+Microsoft 365 common errors
+===========================
+
+**The** :doc:`documentation </channels/microsoft365/index>` **is your
+friend** 😊

```

Comment:
We should add a more meaningful introduction to this page, IMHO.

We can keep the link afterward.